CodeTests/Roles/CodeTest_Roles_1/CodeTest_Roles.cpp

```diff
@@ -110,7 +111,7 @@ int main()
     //Test issue with printing ability out to file
     ofstream nfile;
     nfile.open("Ability.txt");
-    ability->printToFile(file);
+    ability->printToFile(nfile);
```

Comment:
This was a big error that was solved because I kept trying to write to a file that didn't exist lols.